js/app.jsx

```diff
@@ -517,6 +536,7 @@ var Job = React.createClass({
       ['State', state],
       ['Input', inputs],
       ['Output', cleanJobPath(job.conf.output)],
+      ['Scalding Steps', steps],
```

Comment:
Yah, I figured when scalding users see something like: `Scalding Steps: ParquetJob.scala:203`, they're going to be able to infer what it means. Do you want to change it to something like "Scalding Step Line Numbers"? I just thought that might've been a bit long and redundant.